File: backend-module/src/main/java/com/campus/eneity/OperationLog.java
package com.campus.eneity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class OperationLog {
    private int id;
    private String username;
    private int operatorId;
    private String operation;
    private String targetType;
    private int targetId;
    private String detail;
    private String oldData;
    private String newData;
    private String ip;
    private String userAgent;
    private int durationMs;
    private String resultStatus;
    private String failReason;
    private LocalDateTime createdAt;
    private boolean isDeleted;
}

File: frontend/src/types/operationLog.ts
export interface OperationLog {
  id: number
  username: string
  operatorId: number
  operation: string
  targetType: string | null
  targetId: number | null
  detail: string | null
  oldData: string | null
  newData: string | null
  ip: string | null
  userAgent: string | null
  durationMs: number
  resultStatus: string
  failReason: string | null
  createdAt: string
  isDeleted: boolean
}
